README.md

```diff
@@ -84,6 +84,33 @@ Logs to stdout by default:
 $ bin/$(uname -s | tr '[:upper:]' '[:lower:]')/influxd
 ```
 
+### Building with the go command
+
+The `Makefile` provides a wrapper around configuring the utilities for building influxdb. For those wanting to use the `go` command directly, one of two things can be done.
+
+First, the `env` script is located in the root of the repository. This script can be used to execute `go` commands with the appropriate environment configuration.
+
+```bash
+$ ./env go build ./cmd/influxd
+```
+
+Another method is to configure the `pkg-config` utility. Follow the instructions [here](https://github.com/influxdata/flux#getting-started) to install and configure `pkg-config` and then the normal go commands will work.
+
+```bash
+# On Debian/Ubuntu
+$ sudo apt-get install -y clang pkg-config
+# On Mac OS X with Homebrew
```

Comment:
I think breaking this block up into several code blocks will make the steps easier to follow.